### PR TITLE
Release [0.0.30] - 2020-07-08

### PR DESCRIPTION
**Pull request by GiuseppeZeraschi**
# Description

Add active-referral endpoint
  
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made my code self-documenting where possible, with the use of clear variable and function names
- [ ] I have commented the reasons for any complex or unusual code, so others can understand the context
- [ ] I have made corresponding changes to the documentation
- [ ] I have added relevant tests for any new functionality I have added
- [x] I have updated the CHANGELOG
- [x] My changes do not generate any new warnings, either in tooling or in the browser console

### CODE REVIEW

File: src/Interfaces/referral/Referral.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import ReferredUser from './ReferredUser';
+
+export default interface Referral {
+    code: string;
+    totalEarned: number;
+    referredUsers: Array<ReferredUser> | [];
```
Comment (Jamiewarb) [inline src/Interfaces/referral/Referral.ts:6]: Should this be 
```suggestion
    referredUsers: Array<ReferredUser>;
```

`[]` isn't really a type that we use

File: src/Resources/Users/Referral.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import AbstractResource from '../AbstractResource';
+import { HttpClientResponse } from '../../Interfaces/HttpClient';
+
+export default class Referral extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = '';
+        this.defaultAccess = 'member';
+    }
+
+    public get: () => HttpClientResponse = this.createMethodFromPartialSpec({
```
Comment (Jamiewarb) [inline src/Resources/Users/Referral.ts:10]: Looks like a `HttpClientRequest` param type is missing from this

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.30] - 2020-07-08
+### 🚀 Features
+- Add active-referral endpoint
```
Comment (Jamiewarb) [inline CHANGELOG.md:11]: Can you add a link to the original PR for this please, as per the others

File: src/Interfaces/referral/Referral.ts (hunk 1 above)
Comment (GiuseppeZeraschi) [reply src/Interfaces/referral/Referral.ts:6]: I did that because referredUsers can be an empty array...

**Pull request closed without merging**